src/references/interaction_system.md

```diff
@@ -0,0 +1,183 @@
+    </video>
+</div>
+
+### Pointer
+
+Similar to the **Cursor Interaction Module**, the **Pointer Interaction Module** uses the position of the cursor, but for objects in the world, rather than UI. It has the added benefit of being able to detect the angle of intersection with the mouse and the surface of the target object.
+
+#### Pointer Interaction Example
+
+Drag the **Pointer Interaction Template** from **Project Content** into the scene, and start a preview. The cursor will be visible, and the sphere can be interacted by moving the cursor over it, and by clicking.
+
+- Moving the mouse over the cube from any point should change its color using the **Change Color Component**.
+- Clicking the sphere will also change its color using the **Change Color Component**, but also spawns VFX Muzzle Flash that is rotated to show the direction of collision between the pointer and the object, using the **Spawn Object Component** with the **UseInputNormal** and **UseInputPosition** properties enabled.
+
+<!-- TODO: Add video of interaction example -->
+
+### Trigger
+
+The **Trigger Interaction Module** works much like a client-side interactable trigger, but allows for custom keybindings and has more events that can be accessed for that interaction.
+
+!!! info
+    This module will react to Trigger interactions by the local Player or all players depending on the Input Settings. By default the Trigger module will only react to the local Player in order  to act like the other Interaction Modules. If the setting "TriggerLocally" is false then trigger inputs from all players will run. This can be a useful way to mimic networked play without using any actual networking but it could affect your interactions in unexpected ways.
+
+<!-- TODO: Add video of interaction example -->
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/InteractionSystem/InteractionSystem_TriggerExample.mp4" type="video/mp4" />
+    </video>
+</div>
+
+#### Trigger Interaction Example
+
+Drag the **Trigger Interaction Template** from **Project Content** into the scene, and start a preview. Interact with the cube by walking close enough to overlap its trigger, and then clicking.
+
+## Module Events
+
+The Interaction Modules each have several events that can be used as points to broadcast to server scripts, spawn VFX or sound, or anything else that supports individualized design of in-game interactions.
+
+### Events for All Modules
+
+- HoverBegin(position, normal)
+- Hover(position, normal)
+- HoverEnd(position, normal)
+- PressBegin(binding, position, normal)
+- PressEnd(binding, position, normal, wasDragging)
+- Selected()
+- Deselected()
+- SetData(component, ...)
+
+### Events Only for Cursor Interaction Module
+
+- DragOverBegin()
+- DragOverEnd()
+- DragBegin(position, normal)
+- DragEnd(position, normal)
+- Dropped(position, normal, ...)
```

Comment:
probably want all of these names in code tags?

Reply:
Also `Dropped(position, normal, ...)` should be `Dropped(position, normal, dropTargetId, ...)`